src/context.js

```diff
@@ -107,6 +161,31 @@ function EvolvContext(store) {
     return true;
   };
 
+  /**
+   * Persist context key to sessionStorage.
+   *
+   * @param {String} key The key to persist.
+   * @param {boolean} [sessionOnly] If true, the value will only be persisted to sessionStorage.
+   *   Default: true
+   */
+  this.persist = function(key, sessionOnly) {
+    ensureInitialized();
+
+    if (window === undefined || !window.localStorage) {
+      console.log('Evolv: Unable to persist context key. LocalStorage is not available.');
+      return;
+    }
+
+    const keys = sessionOnly === false ? persistedKeys : sessionPersistedKeys;
+    if (!keys.indexOf(key)) {
+      keys.push(key);
+    }
+
+    const local = objects.hasKey(key, localContext);
+    const value = objects.getValueForKey(key, local ? localContext : remoteContext);
+    persistValue(key, value, local, sessionOnly);
+  }
+
```

Comment:
Is it worth having the reverse operation `unpersist()`?

Reply:
I don't think this is necessary.